bin/pasta_file_change_recipients.py

```diff
@@ -0,0 +1,108 @@
+"""
+PaStA - Patch Stack Analysis
+
+Author:
+  Shubhamkumar Pandey <b18194@students.iitmandi.ac.in>
+
+This work is licensed under the terms of the GNU GPL, version 2.  See
+the COPYING file in the top-level directory.
+"""
+
+import os
+import sys
+import argparse
+
+from tqdm import tqdm
+from collections import defaultdict, Counter
+from logging import getLogger
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from pypasta import *
+from pypasta.LinuxMailCharacteristics import email_get_recipients
+from pypasta.Repository.Patch import Diff
+
+log = getLogger(__name__[-15:])
+
+RECIPIENTS = None
```

Comment:
Better pass this variable around than making use of a global variable.

Reply:
Okay